modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -195,6 +195,23 @@ public void testPostDataDefinitionDataLayout() throws Exception {
 			assertValid(postDataLayout);
 		}
 
+		// Data layout with Data Layout Fields (Visual Property)
+
+		DataLayout randomDataLayoutWithVisualProps =
+			_createRandomDataLayoutFieldsWithVisualProps();
+
+		DataLayout postDataLayoutWithVisualProps =
```

Comment:
Can it be just `postDataLayout` ?

Reply:
Yep!

Reply:
Actually, it cannot. `postDataLayout` is already declared in line 190, unless you reuse the same variable (not that I would recommend it).

Reply:
It's ok to reuse because above declaration is inside the `for` scope.  . it's really not reusing but declaring it again

Reply:
I believe that they belong to different scopes, and reusing variables may raise some issues when the codebase grows. It can become cumbersome to realize its current value or reason to be when reading the code. But anyway, if the company standards for naming conventions force us to write it that way, I'm not the one standing against it :sweat_smile: